client/gutenberg/extensions/presets/jetpack/utils/get-jetpack-data.js

```diff
@@ -3,9 +3,23 @@
  * External Dependencies
  */
 import { get } from 'lodash';
+import apiFetch from '@wordpress/api-fetch';
 
 const JETPACK_DATA_PATH = [ 'Jetpack_Editor_Initial_State' ];
 
-export default function getJetpackData() {
-	return get( 'object' === typeof window ? window : null, JETPACK_DATA_PATH, null );
+export default async function getJetpackData() {
+	const jetpackData = get( 'object' === typeof window ? window : null, JETPACK_DATA_PATH, null );
+
+	//we may be on Gutenlypso
+	if ( ! jetpackData ) {
+		const gutenlypsoData = await apiFetch( {
+			path: '/wpcom/v2/gutenberg/available-extensions',
+			method: 'GET',
+			apiNamespace: 'wpcom/v2',
+		} );
+
+		return {
+			available_blocks: gutenlypsoData,
+		};
+	}
```

Comment:
I think here we need to make sure that we return the `jetpackData` if the data is available.